public/scripts/app.js

```diff
@@ -0,0 +1,24 @@
+angular
+  .module("angularBooks", ["ngRoute"])
+  .config(config);
+
+config.$inject = ["$routeProvider", "$locationProvider"]
+
+function config(   $routeProvider,  $locationProvider) {
+  $routeProvider
+    .when("/", {
+      templateUrl: "/templates/books-index.html",
+      controllerAs: "bic",
+      controller: "BooksIndexController"
+    })
+    .when("/books/:id", {
+      templateUrl: "/templates/books-show.html",
+      controllerAs: "bsc",
+      controller: "BooksShowController"
+    })
+
```

Comment:
maybe add a catch all .otherwise here to route users to the homepage if a book doesn't exist or if there is nothing at the path:   .otherwise({ redirectTo: '/' });